app/views/providers/confirm_offices/show.html.erb

```diff
@@ -19,23 +19,22 @@
     <%= govuk_form_group(
           show_error_if: @error,
         ) do %>
+      <fieldset class="govuk-fieldset">
+        <%= govuk_fieldset_header page_title, padding_below: 4 %>
```

Comment:
Question: Is 4 the standard padding_below?  The deleted one was 4
Not a blocker, hopefully a move for standardising!

Reply:
Yep, does the same thing. It was adding a padding below the title. So the govuk_collection_radio_buttons was also creating the page title with a padding below it. It's doing too much 😂